### PR TITLE
Frontend navbar footer

### PR DESCRIPTION
**Pull request by Sankar0415**
changed header and footer for homepage

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,2 +1,4 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<div class = "container" style="min-height:500px">
```
Comment (romain-gille) [inline app/views/pages/home.html.erb]: put in a scss

**Review by romain-gille — COMMENTED** [review]
just change container min height

File: app/views/pages/home.html.erb (hunk 1 above)
Comment (Sankar0415) [reply app/views/pages/home.html.erb]: changed scss. please review it and approve !

**Review by romain-gille — APPROVED** [review]
ok

**Pull request merged**